src/osd/OSD.cc

```diff
@@ -6263,6 +6265,15 @@ void OSD::tick()
     }
   }
 
+  // periodic trim osdmaps
+  {
+    utime_t now = ceph_clock_now();
+    if (now - last_trim_maps > cct->_conf->osd_map_trim_min_interval) {
+      last_trim_maps = now;
+      trim_maps(superblock.cluster_osdmap_trim_lower_bound);
```

Comment:
Not related to the change in the PR, unless I'm missing something, we can remove the parameter from trim_maps entirely and use the superblock's tlb epoch unconditionally.
In the other call to trim_maps (handle_osd_map), the superblock's tlb is updated if necessary beforehand.
@athanatos

Reply:
@jianwei1216, my suggestion is to further cleanup trim_maps(), `trim_maps(epoch_t oldest)` currently gets `oldest` as a parameter. We can remove this parameter and use the `superblock.cluster_osdmap_trim_lower_bound` instead (in a separate commit).

Reply:
@Matan-B already modified, please REVIEW